src/hooks/realm/useRealmConfig/fieldToDisplay/FieldToDisplayConfigOrganization.tsx

```diff
@@ -60,16 +60,39 @@ export default [
 		modifiable: false,
 		tag: 'advanced',
 	},
+	{
+		name: 'Realm',
+		path: 'metadatas.realm',
+		type: 'string',
+		modifiable: false,
+		tag: 'advanced',
+	},
+	{
+		name: 'UserStorage',
+		path: 'metadatas.userStorage',
+		type: 'string',
+		modifiable: false,
+		tag: 'advanced',
+	},
+	{
```

Comment:
I think we should not display it, it's linked to ldap + a certain way to use it that we will hopefully drop soon

Reply:
It's an advanced field so we don't care no ?

Reply:
ok